app_config.py

```diff
@@ -1,5 +1,21 @@
-AUTHORITY = "https://login.microsoftonline.com/Enter_the_Tenant_Name_Here"
+import os
+
+# This pattern is defined in Flask's documentation here
+# https://flask.palletsprojects.com/en/1.1.x/config/#configuring-from-environment-variables
+CLIENT_SECRET = os.getenv("CLIENT_SECRET")
```

Comment:
The app creation scripts still write in config.py, which is deleted ...
.... we don't have a way to automate these apps any longer.

Also , where would customers set the CLIENT_SECRET? on a post-it :) ? this makes life of customes wanting to test the sample a bit harder;

Reply:
The team decision is to settle on still using the placeholder, with inline comment to recommend more secure ways. Resolving for now.